libraries/AP_Math/quaternion.cpp

```diff
@@ -21,6 +21,11 @@
 #include "AP_Math.h"
 #include <AP_InternalError/AP_InternalError.h>
 
+#define HALF_SQRT_2_PlUS_SQRT_2 0.92387953251128673848313610506011 // sqrt(2 + sqrt(2)) / 2
+#define HALF_SQRT_2_MINUS_SQTR_2 0.38268343236508972626808144923416 // sqrt(2 - sqrt(2)) / 2
+#define HALF_SQRT_HALF_TIME_TWO_PLUS_SRT_TWO 0.65328148243818828788676000840496 // sqrt((2 + sqrt(2))/2) / 2
```

Comment:
Static constexpr ftype ?

Reply:
These are a define because the existing `HALF_SQRT_2` was, I'm not really sure the benefit either way.